pkg/source/ttnv3/util.go

```diff
@@ -59,7 +59,6 @@ func clearDeviceSession(dev *ttnpb.EndDevice) error {
 	return dev.SetFields(nil,
 		"activated_at",
 		"mac_state",
-		"last_dev_status_f_cnt_up",
```

Comment:
Why this change ?

Reply:
`last_dev_status_f_cnt_up` is a part of `mac_state`. Keeping this line resulted in an error:
![image](https://user-images.githubusercontent.com/73070465/222161739-554bb46c-b80f-4e0a-affa-fddbf4c4a25a.png)